README.rst

```diff
@@ -157,6 +157,25 @@ Available optimization problems
      - |Build Status ResNetClassif|
 
 
+Citing Benchopt
+---------------
+
+If you use ``Benchopt`` in scientific publication, please consider citing the following paper
+
+.. code-block:: bibtex
+
+   @article{benchopt,
+      author = {Thomas Moreau, Mathurin Massias, Alexandre Gramfort, Pierre Ablin,
+                Pierre-Antoine Bannier, Benjamin Charlier, Mathieu Dagréou, Tom Dupré la Tour,
```

Comment:
bibtex may have difficulty splitting between first and last names in this format, you can use Moreau, Thomas and Massias, Mathurin and ... etc